include/swift/SILOptimizer/Utils/InstOptUtils.h

```diff
@@ -375,7 +387,42 @@ bool tryCheckedCastBrJumpThreading(
 /// cleaning up any dead code resulting from deleting those instructions. Use
 /// this utility instead of
 /// \c recursivelyDeleteTriviallyDeadInstruction.
+///
```

Comment:
Added comment
```
/// this utility instead of \p recursivelyDeleteTriviallyDeadInstruction
/// as follows:
///   InstructionDeleter deleter;
///   deleter.deleteIfDead();
///   deleter.cleanupDeadInstructions();
```